### DISCUSSION TITLE
Would it be beneficial to use https://github.com/knadh/koanf?

### DISCUSSION THREAD

**Discussion by bradenhilton** [twpayne/chezmoi#2786] [Ideas]
I just came across this in my explore section and thought it looked interesting. But I'm unsure if it would be worth switching to or not.

**Comment by twpayne** [comment]
I don't think so. I did look at this when replacing Viper. The config file reading is only a few lines of code, and I don't think we need to pull in all the dependencies that knadh/koanf adds.
